public/app/plugins/panel/text/TextPanel.tsx

```diff
@@ -41,24 +41,34 @@ export class TextPanel extends PureComponent<Props, State> {
   }
 
   prepareHTML(html: string): string {
-    return this.interpolateAndSanitizeString(html);
+    const result = this.interpolateString(html);
```

Comment:
🤔 Nit: Can we change the name of this function to something more meaningful, maybe `interpolateVariable`?

Reply:
yeah, `Variables` :)